app/parse.py

```diff
@@ -1,10 +1,32 @@
-from dataclasses import dataclass
+import csv
+from dataclasses import dataclass, fields, astuple
 from urllib.parse import urljoin
 
+from bs4 import BeautifulSoup
+from selenium.common.exceptions import (
+    TimeoutException,
+    NoSuchElementException,
+    ElementClickInterceptedException,
+    ElementNotInteractableException
+)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from app.chrome_driver import ChromeDriver
 
 BASE_URL = "https://webscraper.io/"
 HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
 
+ENDPOINTS_TO_PARSE = {
+    "home": "",
+    "computers": "computers",
+    "laptops": "computers/laptops",
+    "tablets": "computers/tablets",
+    "phones": "phones",
+    "touch": "phones/touch",
+}
+
```

Comment:
let's don't overwhelm this file. Create a separate file for constants, dataclasses, classes, etc